google/cloud/aiplatform/training_utils/environment_variables.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import json
+import os
+
+from typing import Dict, Optional
+
+
+class EnvironmentVariables:
```

Comment:
I think it may slightly less cumbersome to make these environment variables module level attributes. Right now the usage requires instantiating `EnvironmentVariables`:

```Python
from google.cloud import training_utils

env_var = training_utils.EnvironmentVariables()
env_var.training_data_uri
```

If may be more straight forward to do this:
```Python
from google.cloud.training_utils import environment_variables

environment_variables.training_data_uri
```

After the initial import of `os`, `environ` doesn't reflect any new changes unless `environ` is changed directly: https://docs.python.org/3/library/os.html#os.environ

If we envision changing `environ` as part of the implementation in the future then we should take the class based approach but if not it will be less lines of code to go with module attributes.

What are you thoughts?

Reply:
I've updated the variables to be module level attributes, let me know your thoughts.